actors/stage/src/main/java/com/ea/orbit/actors/Stage.java

```diff
@@ -404,4 +414,40 @@ public void bind()
     {
         return execution.getState();
     }
+
+    private void startMetrics()
+    {
+        try
+        {
+            Class mmClazz = Class.forName("com.ea.orbit.metrics.MetricsManager"); //make sure the metrics manager is on the classpath.
+            Method getInstanceMethod = mmClazz.getDeclaredMethod("getInstance");
+            Method initializeMetricsMethod = mmClazz.getDeclaredMethod("initializeMetrics", List.class);
+            Method registerExportedMetricsMethod = mmClazz.getDeclaredMethod("registerExportedMetrics", Object.class, String.class);
+
+            Object managerObject = getInstanceMethod.invoke(null, null);
+            initializeMetricsMethod.invoke(managerObject, metricsConfig);
+            registerExportedMetricsMethod.invoke(managerObject, execution, execution.runtimeIdentity());
+        }
+        catch (Exception ex)
```

Comment:
instead of catching exception how about two catches, one for class not found == metrics isn't being used and another to catch other errors == log error.